README.md

```diff
@@ -1,11 +1,18 @@
 # Documentazione
 
-Repository principale per tutta la documentazione di ogni cosa presente al PHC, ogni progetto corrisponde ad una cartella
+Repository principale per tutta la documentazione di ogni cosa presente al PHC,
+ogni progetto corrisponde ad una cartella
 
-## [Lista delle Macchine](./macchine)
+## Macchine
```

Comment:
Secondo me i link si possono anche lasciare, li trovavo molto comodi per navigare nel repo da GitHub

Reply:
Li ho rimossi perché non piacevano all'impaginatore, ma effettivamente non ho capito perché.
Sarà una cosa di mdbook?

Reply:
Mmh, forse genera il table of contents e si confonde

Reply:
Almeno io pensavo fosse più comodo leggere direttamente da Github, tu invece pensavi di renderizzarlo con mdbook?